plugins/outputs/wavefront/wavefront.go

```diff
@@ -142,7 +142,7 @@ func (w *Wavefront) Connect() error {
 			FlushIntervalSeconds: flushSeconds,
 		})
 		if err != nil {
-			return fmt.Errorf("Wavefront: Could not create Wavefront Sender for Url: %s", w.Url)
+			return fmt.Errorf("wavefront: Could not create Wavefront Sender for Url: %s", w.Url)
```

Comment:
Wavefront is a proper name and shouldn't be lowercased, but it's also not necessary as the logger will tag the plugin name elsewhere

Reply:
`wavefront: ` removed from message - logger will tag the plugin name elsewhere